terraform-module/modules/frontend-spa-cdn/main.tf

```
@@ -148,6 +142,36 @@ resource "aws_cloudfront_distribution" "this" {
   }
 }
 
+resource "aws_cloudwatch_log_delivery_source" "cloudfront_log_source" {
+  region = "eu-west-1"
+
+  name         = "cloudfront_log_source"
```

the resource is `log_delivery_source` so no need to repeat it in the name
```suggestion
  name         = "cloudfront"
```